src/i18n/strings/fr.json

```diff
@@ -128,7 +128,7 @@
     "Displays action": "Affiche l'action",
     "Don't send typing notifications": "Ne pas envoyer les notifications de saisie",
     "Download %(text)s": "Télécharger %(text)s",
-    "Drop here %(toAction)s": "Déposer ici %(toAction)s",
+    "Drop here %(toAction)s": "Déposer ici pour %(toAction)s",
```

Comment:
@jeangb this looks wrong - the english doesn't have 'to' in it, so why would the french have 'pour'?